### PR TITLE
AT_17.04.003 | Move Organization Folder

### PR DESCRIPTION
**Pull request by nadya0101**
https://trello.com/c/vn4YPtRZ/1232-at1704003-move-organization-folder

### CODE REVIEW

File: cypress/e2e/moveOgranizationFolder.cy.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/// <reference types="cypress"/
+
+import names from "../fixtures/create2folders.json"
+
+beforeEach('create2folders', function () {
+
+  cy.get('[href="/view/all/newJob"]').click()
+  cy.get('#name').type(names.orgFolderName)
+  cy.contains('Organization Folder').click()
```
Comment (OLgaByR) [inline cypress/e2e/moveOgranizationFolder.cy.js:9]: 'Organization Folder' should be moved to fixtures

File: cypress/e2e/moveOgranizationFolder.cy.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  cy.get('#name').type(names.folderName)
+  cy.get('#ok-button').click()
+  cy.get('#jenkins-home-link').click();
+})
+
+describe('Move Organization Folder', () => {
+    it('AT_17.04.003 | Move Organization Folder', function () {
+      cy.get('.jenkins-table__link').contains(names.orgFolderName).realHover();
+      cy.get('[href="job/OrgFolder/"]').click();
+      cy.contains('Move').click()
```
Comment (OLgaByR) [inline cypress/e2e/moveOgranizationFolder.cy.js:24]: text in contains should be moved to fixtures

File: cypress/e2e/moveOgranizationFolder.cy.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      cy.get('[name="destination"]').realHover().select('/Folder');
```
Comment (OLgaByR) [inline cypress/e2e/moveOgranizationFolder.cy.js:25]: text in select should be moved to fixtures

File: cypress/fixtures/create2folders.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "orgFolderName": "OrgFolder",
+
```
Comment (OLgaByR) [inline cypress/fixtures/create2folders.json:3]: excessive line 3, should be deleted

File: cypress/fixtures/create2folders.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+    "orgFolderName": "OrgFolder",
+  
+      "folderName": "Folder"
+  }
```
[the remark this answers is not in the document]
Comment (EkBlinova) [reply cypress/fixtures/create2folders.json:12]: Nadya, I agree, you can use any existing files in the fixtures: createFolder.json and organizationFolderNames.json

**Review by EkBlinova — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Pull request merged**